src/plugins/intel_cpu/src/nodes/executors/dnnl/dnnl_convolution_primitive.cpp

```diff
@@ -871,7 +871,7 @@ std::shared_ptr<DnnlConvolutionPrimitive> DnnlConvolutionPrimitive::create(
     auto runtimeCache = context->getRuntimeCache();
     const auto result = runtimeCache->getOrCreate(dnnlConvKey, builder);
     const auto& primitive = result.first;
-    assert(primitive);
+    OPENVINO_DEBUG_ASSERT(primitive, "primitive is not valid");
```

Comment:
failed to create a DnnlConvolutionPrimitive

Reply:
Done